zenodo/modules/deposit/jsonschemas/deposits/records/record-v1.0.0.json

```diff
@@ -940,6 +940,29 @@
     "version": {
       "description": "Record version tag.",
       "type": "string"
+    },
```

Comment:
Remember to change these in `base-v1.0.0.json` and then run the `zenodo jsonschema compile` commands